docs/group_faqs/gf_multiple_account.rst

```diff
@@ -0,0 +1,31 @@
+Solace Cloud Multiple Account
+=============================
+
+Overview
+~~~~~~~~
+Users can be added to more than one Solace Cloud Account. Once added, a user can have different access and permission in each environment.
+
+For example, an organization may purchase two different subscriptions for (production, non-production, etc.) environments. The organization will be able to add users to one or more environments, enabling them to segregate their user base e.g. contractors, sales engineers, and different profession service groups.
```

Comment:
I don't understand how an organization purchasing different subscriptions has anything to do with a user having different access/permissions in different accounts. By leading with 'for example' you're saying that it's an example for your previous point.